server/api/log_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/log"
+	"github.com/pingcap/pd/v4/server"
+)
+
+var _ = Suite(&testLogSuite{})
+
+type testLogSuite struct {
+	svr       *server.Server
+	cleanup   cleanUpFunc
+	urlPrefix string
+}
+
+func (s *testLogSuite) SetUpSuite(c *C) {
+	s.svr, s.cleanup = mustNewServer(c)
+	mustWaitLeader(c, []*server.Server{s.svr})
+
+	addr := s.svr.GetAddr()
+	s.urlPrefix = fmt.Sprintf("%s%s/api/v1/admin", addr, apiPrefix)
+
+	mustBootstrapCluster(c, s.svr)
+}
+
+func (s *testLogSuite) TearDownSuite(c *C) {
+	s.cleanup()
+}
+
+func (s *testLogSuite) TestSetLogLevel(c *C) {
+	level := "error"
+	if log.GetLevel().String() == level {
```

Comment:
I think in this test, it won't execute line 49?

Reply:
If the original level is error. It‘s uncertain whether setloglevel is executed correctly.

Reply:
> I think in this test, it won't execute line 49?
If the original level is error. It‘s uncertain whether setloglevel is executed correctly.

Reply:
I mean in this test, `log.GetLevel().String()` will be `info`, this if statement will always be false?